modules/serverless-logic-persistence-db-schema-initialization.adoc

```diff
@@ -0,0 +1,104 @@
+// Module included in the following assemblies:
+// * serverless-logic/serverless-logic-managing-persistence
+
+
+:_mod-docs-content-type: REFERENCE
+[id="serverless-logic-persistence-db-schema-initialization_{context}"]
+= Database schema initialization
+
+When you are using `SonataFlow` with PostgreSQL persistence, you can initialize the database schema either by enabling Flyway or by manually applying database schema updates using DDL (Data Definition Language) scripts.
+
+Flyway is managed by `kie-addons-quarkus-flyway` runtime module and it is disabled by default. To enable Flyway, you must configure it using one of the following methods:
+
+[id="serverless-logic-flyway-config-workflow-configmap_{context}"]
+== Flyway configuration in the workflow ConfigMap
+
+To enable Flyway in the workflow ConfigMap, you can add the following property:
+
+.Example of enabling Flyway in the workflow ConfigMap
+[source,yaml]
+----
+apiVersion: v1  
+kind: ConfigMap  
+metadata:  
+  labels:  
+    app: example-workflow  
+  name: example-workflow-props  
+data:  
+  application.properties: |  
+    kie.flyway.enabled = true
+----
+
+[id="serverless-logic-flyway-config-environment-variables_{context}"]
+== Flyway configuration using environment variables in the Workflow Container
+
+You can enable Flyway by adding an environment variable to the `spec.podTemplate.container` field in the `SonataFlow` CR by using the following example:
+
+.Example of enabling Flyway by using the workflow container environment variable
+[source,yaml]
+----
+apiVersion: sonataflow.org/v1alpha08  
+kind: SonataFlow  
+metadata:  
+  name: example-workflow  
+  annotations:  
+    sonataflow.org/description: Example Workflow  
+    sonataflow.org/version: 0.0.1  
+spec:  
+  podTemplate:  
+    container:  
+      env:  
+        - name: KIE_FLYWAY_ENABLED  
+          value: 'true'  
+  flow: ...
+----
+
+[id="serverless-logic-flyway-config-sonataflowplatform-properties_{context}"]
+== Flyway configuration using SonataFlowPlatform properties
+
+To apply a common Flyway configuration to all workflows within a namespace, you can add the property to the `spec.properties.flow` field of the `SonataFlowPlatform` CR shown in the following example:
+
+[NOTE]
+====
+This configuration is applied during workflow deployment. Ensure the Flyway property is set before deploying workflows.
+====
+
+.Example of enabling Flyway by using the `SonataFlowPlatform` properties
+[source,yaml]
+----
+apiVersion: sonataflow.org/v1alpha08  
+kind: SonataFlowPlatform  
+metadata:  
+  name: sonataflow-platform  
+spec:  
+  properties:  
+    flow:  
+      - name: kie.flyway.enabled  
+        value: true
+----
+
+[id="serverless-logic-manual-db-initialization-using-ddl_{context}"]
+== Manual database initialization using DDL scripts
```

Comment:
This section has a procedure and should not be contained within a reference module, this should be pulled out into its own procedure module. It should also have the following characteristics:

- A heading that has a gerund in it (example: **Initializing** a manual database using DDL scripts)
- a .Procedure title right above the steps

Reply:
Done!